graphql/schemabuilder/types.go

```diff
@@ -61,6 +61,16 @@ func (s *Object) FieldFunc(name string, f interface{}, options ...FieldFuncOptio
 	s.Methods[name] = m
 }
 
+// Key registers the key field on an object. The field should be specified with the initial
```

Comment:
how about "should be specified by the name of the graphql field, i.e. camelCased for struct fields"